dicodile/utils/csc.py

```diff
@@ -155,7 +155,7 @@ def _dense_convolve_multi_uv(z_hat, uv):
     return Xi
 
 
-def _dense_transpose_convolve(residual_i, D=None, n_channels=None):
+def dense_transpose_convolve(residual_i, D):
```

Comment:
I think I would rather keep this one private:
```suggestion
def _dense_transpose_convolve(residual_i, D):
```